.travis.yml

```
@@ -11,23 +11,40 @@ branches:
   only:
   - master
 
-# install proper NodeJS for bids-validator 
 before_install:
+  # install spine-generic for acquisition parameters check
+  # TODO: we should probably version-pin this?
+  - pip install spine-generic@git+https://github.com/kousu/spine-generic.git@patch-1
+
+  # use conda's git-annex, because it's the most up to date;
+  # Debian's is very behind, and even neurodebian's is behind
+  # by a year, and is currently incompatible with the format of this repo.
+  - wget -O/tmp/Miniconda3-latest-Linux-x86_64.sh https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+  - bash /tmp/Miniconda3-latest-Linux-x86_64.sh -b -f
+  - . ~/miniconda3/bin/activate # careful! after this we're no longer in Travis's python, we're in conda's!
+  - conda install -y -c conda-forge git-annex
```

Maybe, to avoid having Travis' python and conda's collide, it would be better to isolate it like this:

```suggestion
  - export PATH=$PATH:~/miniconda3/bin
  - (. ~/miniconda3/bin/activate && conda install -y -c conda-forge git-annex)
```